### PR TITLE
Add postgresql database support

### PR DESCRIPTION
**Pull request by clcc2019**

### CODE REVIEW

**Review by gaius-qi — CHANGES_REQUESTED** [review]
@clcc2019 Please add changelog and update chart version, refer to https://github.com/dragonflyoss/helm-charts/pull/190/files.

Helm charts use helm-docs to generate helm documents, refer to https://github.com/dragonflyoss/helm-charts/tree/main/charts.

Remove this.
image

File: charts/dragonfly/values.yaml [hunk 1]
```diff
@@ -1488,6 +1488,44 @@ externalMysql:
...
+  # -- External postgresql username.
+  username: dragonfly
+  # -- External postgresql password.
+  password: dragonfly
+  # -- External postgresql database name.
+  database: manager
+  # -- External postgresql port.
+  port: 5432
+  
+
```
Comment (gaius-qi) [inline charts/dragonfly/values.yaml:1528]: Remove blank lines.

**Review by gaius-qi — CHANGES_REQUESTED** [review]

**Comment by gaius-qi** [comment]
@clcc2019 Please rebase main branch.

File: charts/dragonfly/Chart.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: dragonfly
 description: Dragonfly is an intelligent P2P based image and file distribution system
 icon: https://raw.githubusercontent.com/dragonflyoss/Dragonfly2/main/docs/images/logo/dragonfly.svg
 type: application
-version: 1.1.9
+version: 1.2.0
```
Comment (gaius-qi) [inline charts/dragonfly/Chart.yaml:6]: version: 1.1.10

File: charts/dragonfly/Chart.yaml [hunk 3]
```diff
@@ -26,8 +26,8 @@ sources:
 
 annotations:
   artifacthub.io/changes: |
-    - Add syncPeers configuration to manager.
-    - Update dragonfly image tag to v2.1.16.
+    - Add postgresql database
```
Comment (gaius-qi) [inline charts/dragonfly/Chart.yaml:29]: `- Add postgresql database config.`

File: charts/dragonfly/Chart.yaml [hunk 4]
```diff
@@ -26,8 +26,8 @@ sources:
...
+    - Fix dfdaemon seedPeer configmap
```
Comment (gaius-qi) [inline charts/dragonfly/Chart.yaml:30]: `- Fix dfdaemon seedPeer configmap.`

**Review by gaius-qi — CHANGES_REQUESTED** [review]
@clcc2019  Remove blank line.

image

**Comment by gaius-qi** [comment]
@clcc2019 Please fix lint.

**Review by gaius-qi — CHANGES_REQUESTED** [review]
Fix lint.

**Comment by teszes** [comment]
@clcc2019 Are you still working on this? This would be useful to me too, and it seems there is only some lint errors to fix.

**Comment by clcc2019** [comment]
> Fix lint.

Can I skip the bitnami charts package check?

**Comment by gaius-qi** [comment]
> > Fix lint.
> 
> Can I skip the bitnami charts package check?

I don't think it's good.

**Comment by gaius-qi** [comment]
@clcc2019 Close PR due to no progress for a long time

**Comment by lucasfcnunes** [comment]
@clcc2019 this PR is a great idea. I am really willing to take over and finish your work.

**Pull request closed without merging**